# Remove cov options from pytest.ini
--- a/pytest.ini
+++ b/pytest.ini
@@ -29,16 +29,16 @@
 
   # `pytest-cov`:
   # `pytest-cov`, "-p" preloads the module early:
-  -p pytest_cov
-  --no-cov-on-fail
-  --cov=cherrypy
-  --cov-branch
-  --cov-report=term-missing:skip-covered
-  --cov-report=html:.tox/tmp/test-results/pytest/cov/
-  --cov-report=xml
+#  -p pytest_cov
+#  --no-cov-on-fail
+#  --cov=cherrypy
+#  --cov-branch
+#  --cov-report=term-missing:skip-covered
+#  --cov-report=html:.tox/tmp/test-results/pytest/cov/
+#  --cov-report=xml
   # --cov-report xml:.test-results/pytest/cov.xml  # alternatively move it here
-  --cov-context=test
-  --cov-config=.coveragerc
+#  --cov-context=test
+#  --cov-config=.coveragerc
 
 doctest_optionflags = ALLOW_UNICODE ELLIPSIS
 
