# Remove cov options from pytest.ini
--- cherrypy3-18.6.0.orig/pytest.ini
+++ cherrypy3-18.6.0/pytest.ini
@@ -25,11 +25,6 @@ addopts =
     # dump the test results in junit format:
     --junitxml=.test-results/pytest/results.xml
 
-    # `pytest-cov`:
-    --cov=cherrypy
-    --cov-report term-missing:skip-covered
-    --cov-report xml
-    # --cov-report xml:.test-results/pytest/cov.xml  # alternatively move it here
 doctest_optionflags = ALLOW_UNICODE ELLIPSIS
 filterwarnings =
     error
